### DISCUSSION TITLE
emmc chip replacement

### DISCUSSION THREAD

**Discussion by chuckkay** [OpenNeptune3D/OpenNept4une#218] [Q&A]
the 7gb emmc is way to small and 80% full without any gcodes files, and apparently its not a larger emmc because i tried the expand partition feature and still only show 6.9GB.... please provide answers to these questions for swapping it out with a 32gb emmc chip:

1- what step by step process should i follow after removing the old chip, in order to transfer everything to the new chip without loosing bootability and maintaining settings...  or do i need to backup my klipper config files and install and flash openneptune from scratch?

2 - is this an acceptable model EMMC  https://a.co/d/avzAQqN

**Comment by steve8x8** [comment]
2 - looks similar to the one I got from Ali
1 - since a fresh install would just copy a disk image over the beginning of a (possibly bigger) EMMC, you should be able to copy your old EMMC to a bigger one - I'd prefer a Linux platform and matching USB or microSD adapters for the cards.
